src/bin/patchmanager-daemon/patchmanagerobject.h

```diff
@@ -73,6 +74,51 @@ class QSettings;
 class QNetworkAccessManager;
 class PatchManagerAdaptor;
 class QLocalServer;
+
+class PatchManagerFilter : public QObject, public QCache<QString, QObject>
+{
+    Q_OBJECT
+    Q_PROPERTY(bool active READ active WRITE setActive NOTIFY activeChanged)
+    Q_PROPERTY(unsigned int hits READ hits)
+    Q_PROPERTY(unsigned int misses READ misses)
+public:
+    PatchManagerFilter(QObject *parent = nullptr, int maxCost = 100);
+    //~PatchManagerFilter();
+
+    void setup();
+    // override QCache::contains()
+    bool contains(const QString &key) const
+    {
+       if (m_active) {
```

Comment:
```suggestion
       if (!m_active) {
```

Reply:
Thanks for confirming that one. I came across it and it looked wrong but didn't have the time/mind to go over it.